CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -28,40 +32,80 @@ local hopperbinGUI = Instance.new("SurfaceGui", CoreGui)
 hopperbinGUI.Adornee = hopperbinPart
 hopperbinGUI.ToolPunchThroughDistance = 1000
 hopperbinGUI.Name = "HopperBinGUI"
+local toolsFrame = Instance.new("Frame", hopperbinGUI)
+toolsFrame.Size = UDim2.new(1, 0, 0, ICON_SIZE)
+toolsFrame.Position = UDim2.new(0, 0, 0, HEALTHBAR_SPACE)
+toolsFrame.BackgroundTransparency = 1
+
+local healthbarBack = Instance.new("Frame", hopperbinGUI)
+healthbarBack.BackgroundColor3 = Color3.new(1, 1, 1)
+healthbarBack.BorderSizePixel = 0
+healthbarBack.Name = "HealthbarBack"
+local healthbarFront = Instance.new("Frame", healthbarBack)
+healthbarFront.BorderSizePixel = 0
+healthbarFront.Size = UDim2.new(1, 0, 1, 0)
+healthbarFront.Position = UDim2.new(0, 0, 0, 0)
+healthbarFront.BackgroundColor3 = Color3.new(0, 1, 0)
+healthbarFront.Name = "HealthbarFront"
+
+local greenColor = Color3.new(0.2, 1, 0.2)
+local redColor = Color3.new(1, 0.2, 0.2)
+local yellowColor = Color3.new(1, 1, 0.2)
```

Comment:
Since these variables are constants we should keep them consistent with our naming style such as: `GREEN_COLOR`